GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.js

```diff
@@ -287,7 +352,13 @@ gdjs.WebsocketDebuggerClient.prototype._circularSafeStringify = function(
   );
 };
 
-// JSON serializer that prevent circular references and stop if maxDepth is reached.
+/** 
+ * Generates a JSON serializer that prevent circular references and stop if maxDepth is reached.
+ * @param {Function} [replacer] - The value to replace unsafe values?
```

Comment:
Let's also use `A function called for each property on the object or array being stringified, with the property key and its value, and that returns the new value. If not specified, values are not altered.`